scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -16,6 +16,20 @@ namespace <%= namespace %> {
     public class NoMatchingVariantException : Exception {
         public NoMatchingVariantException(string message) : base(message) {}
     }
+
+    /// <summary>
+    /// Wraps around a shipping address and shipping identifier. Used for sending the final checkout fields in one GraphQL query.
+    /// </summary>
+    public struct ShippingFields {
+        public MailingAddressInput ShippingAddress;
```

Comment:
I wonder if it should exist in `iOSNativeCheckout` and `SetFinalCheckoutFields` should also live there also.